core/editor/src/dispatcher/events.rs

```diff
@@ -17,92 +24,80 @@ pub enum Response {
 }
 
 #[derive(Debug, Clone, Default)]
-pub struct Trace(Vec<MouseState>);
+pub struct Trace(Vec<TracePoint>);
+
+impl Deref for Trace {
+	type Target = Vec<TracePoint>;
```

Comment:
Shouldn't this be used by the struct definition above?

Reply:
Could you rephrase that? I don't really understand the question.